build_info/bottom-rs.control

```diff
@@ -0,0 +1,7 @@
+Package: bottom-rs
+Version: @DEB_BOTTOM-RS_V@
+Architecture: @DEB_ARCH@
+Maintainer: @DEB_MAINTAINER@
```

Comment:
Add the Author field with ``bottom-software-foundation, Kaylynn`` below this line.